development.md

```diff
@@ -31,11 +31,21 @@ Note: before running the script, ensure that through the CLI, you are logged int
 
 After confirming that your changes pass local testing:
 1. Clean up testing artifacts
-2. Create a PR. Once your PR has been merged, a Github Actions workflow (Docker Publish) will be triggered, to build, tag (with `stable`), and push
-   all example images. This ensures that all example images are using the most up-to-date version of the SDK, i.e. the one including your
-   changes.
+2. Create a PR. Once your PR has been merged, a Github Actions workflow (`Docker Publish`) will be triggered, to build, tag (with `stable`), and push
+all example images. This ensures that all example images are using the most up-to-date version of the SDK, i.e. the one including your changes
+3. If your SDK changes included modifications to any files in `pkg/info` or `pkg/apis/proto`, then get the commit SHA
```

Comment:
Good catch. nit: For 3, please explain why first and then how, most of go sdk developers know how. Same for below.